index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <img class="background-image" src="images/header-shapes-mobile.png" alt="background image">
+      <div class="box1">
+        Welcome  
+      </div>
+      <div class="box2">
+        <i class="fa fa-bars"></i>
+      </div>
+      <div class="onscreen">
+          <h1>I'm Amanulla Glad to see you!</h1>
+      </div>
```

Comment:
- [ ] I guess you are missing the content here, below your main title. try to check on your design and figma design, there is a bit different, and you are supposed to have something closer or similar.  Please add the the content based on proved design.




Your page| Figma design
--- | ---
![image](https://user-images.githubusercontent.com/43877689/170440562-5efc0c82-aeeb-41cd-bdac-17804bd57ee0.png) | ![image](https://user-images.githubusercontent.com/43877689/170440687-1378f571-9670-4cd8-8745-6d484459e6ed.png)